static/templates/block_external.js

```diff
@@ -0,0 +1,17 @@
+  function capture(e) { $(e.target).attr("href", encodeURI("/external?source=" + e.target.href)); }
+  jk( document ).ready(function() {
+    jk("a").on({click: function(e) {
+      if ("target" in e && "href" in e.target) {
+        var href = e.target.href;
+        if (href.indexOf(window.location.origin) == 0)
+          return;
+        if (href.substr(0, 2) == "//")
+          return capture(e);
+        if (href.substr(0, 5) == "http:")
+          return capture(e);
+        if (href.substr(0, 6) == "https:")
+          return capture(e);
+        return;
+      }
+    }});
+  });
```

Comment:
I wonder if the js could check if the current url start with "external", if this case, it would not capture links.

So, we would have to set `blockExternalLink` at response level. It would be only a server configuration (introduce this js or not).

It would be pretty useful if we do this later (https://github.com/kiwix/kiwix-lib/issues/322#issuecomment-584225014). As the inserted js would be always present, it would not be possible to set a boolean to avoid the block of the links. And the boolean in the response would be useless.